omnibus/config/software/system-probe.rb

```diff
@@ -62,4 +61,6 @@
   end
 
   copy 'pkg/ebpf/c/COPYING', "#{install_dir}/embedded/share/system-probe/ebpf/"
+
+  link "#{install_dir}/bin/agent/agent", "#{install_dir}/embedded/bin/system-probe"
```

Comment:
Do you need this ? It feels like something that is a remaining of the "one-binary" work

Reply:
Oups., good catch.